src/components/MessagesList/MessagesList.vue

```diff
@@ -739,20 +764,9 @@ export default {
 				if (exception?.response?.status === 304) {
 					// 304 - Not modified
 					debugTimer.end(`${this.token} | fetch history`, 'status 304')
-					this.stopFetchingOldMessages = true
```

Comment:
Ah yes, I remember it was put there for that case specifically rather than just relying on first message. I will keep the flag here and add it to computed